<ui version="4.0" stdsetdef="1" >
  <author></author>
  <comment></comment>
  <exportmacro></exportmacro>
  <class>TGo4BrowserOptions</class>
  <widget class="QGo4Widget" name="TGo4BrowserOptions" >
    <property name="geometry" >
      <rect>
        <x>0</x>
        <y>0</y>
        <width>243</width>
        <height>29</height>
      </rect>
    </property>
    <property name="minimumSize" >
      <size>
        <width>0</width>
        <height>0</height>
      </size>
    </property>
    <property name="windowTitle" >
      <string>Browser options</string>
    </property>
    <layout class="QHBoxLayout" >
      <property name="margin" >
        <number>1</number>
      </property>
      <property name="spacing" >
        <number>1</number>
      </property>
      <item>
        <widget class="QToolButton" name="RefreshBtn" >
          <property name="text" >
            <string/>
          </property>
          <property name="icon" >
            <iconset>:/icons/refresh.png</iconset>
          </property>
          <property name="autoRaise" >
            <bool>true</bool>
          </property>
          <property name="toolTip" stdset="0" >
            <string>Update browser content (request list of objecs from analysis)</string>
          </property>
        </widget>
      </item>
      <item>
        <widget class="QToolButton" name="StartMonitorBtn" >
          <property name="text" >
            <string/>
          </property>
          <property name="icon" >
            <iconset>:/icons/startselected.png</iconset>
          </property>
          <property name="autoRaise" >
            <bool>true</bool>
          </property>
          <property name="toolTip" stdset="0" >
            <string>Start monitoring of visible objects</string>
          </property>
        </widget>
      </item>
      <item>
        <widget class="QSpinBox" name="IntervalSpin" >
          <property name="minimumSize" >
            <size>
              <width>50</width>
              <height>0</height>
            </size>
          </property>
          <property name="suffix" >
            <string> s</string>
          </property>
          <property name="maximum" >
            <number>100</number>
          </property>
          <property name="minimum" >
            <number>2</number>
          </property>
          <property name="singleStep" >
            <number>2</number>
          </property>
          <property name="toolTip" stdset="0" >
            <string>Monitoring interval</string>
          </property>
        </widget>
      </item>
      <item>
        <widget class="QToolButton" name="StopMonitorBtn" >
          <property name="text" >
            <string/>
          </property>
          <property name="icon" >
            <iconset>:/icons/Stop.png</iconset>
          </property>
          <property name="autoRaise" >
            <bool>true</bool>
          </property>
          <property name="toolTip" stdset="0" >
            <string>Stop monitoring</string>
          </property>
        </widget>
      </item>
      <item>
        <widget class="QComboBox" name="FilterBox" >
          <property name="toolTip" stdset="0" >
            <string>Select type of items, visible in browser</string>
          </property>
          <item>
            <property name="text" >
              <string>All items</string>
            </property>
          </item>
          <item>
            <property name="text" >
              <string>Fetched</string>
            </property>
          </item>
          <item>
            <property name="text" >
              <string>Monitored</string>
            </property>
          </item>
        </widget>
      </item>
      <item>
        <widget class="QToolButton" name="ClearBtn" >
          <property name="text" >
            <string/>
          </property>
          <property name="icon" >
            <iconset>:/icons/clear.png</iconset>
          </property>
          <property name="autoRaise" >
            <bool>true</bool>
          </property>
          <property name="toolTip" stdset="0" >
            <string>Clear histograms and conditions in analysis</string>
          </property>
        </widget>
      </item>
    </layout>
  </widget>
  <layoutdefault spacing="0" margin="0" />
  <includes>
    <include location="local">QGo4Widget.h</include>
  </includes>
  <connections>
    <connection>
      <sender>RefreshBtn</sender>
      <signal>clicked()</signal>
      <receiver>TGo4BrowserOptions</receiver>
      <slot>RefreshBtn_clicked()</slot>
    </connection>
    <connection>
      <sender>ClearBtn</sender>
      <signal>clicked()</signal>
      <receiver>TGo4BrowserOptions</receiver>
      <slot>ClearBtn_clicked()</slot>
    </connection>
    <connection>
      <sender>StartMonitorBtn</sender>
      <signal>clicked()</signal>
      <receiver>TGo4BrowserOptions</receiver>
      <slot>StartMonitorBtn_clicked()</slot>
    </connection>
    <connection>
      <sender>StopMonitorBtn</sender>
      <signal>clicked()</signal>
      <receiver>TGo4BrowserOptions</receiver>
      <slot>StopMonitorBtn_clicked()</slot>
    </connection>
    <connection>
      <sender>FilterBox</sender>
      <signal>activated(int)</signal>
      <receiver>TGo4BrowserOptions</receiver>
      <slot>FilterBox_activated(int)</slot>
    </connection>
  </connections>
</ui>
